quicktests/exampleUtil.js

```diff
@@ -109,3 +109,10 @@ function generateHeightWeightData(n) {
 
   return data;
 }
+
+var deep_copy = function(from, to){
+  var deep_copy_xy = function(d){
+    to.push({'x': d.x, 'y': d.y});
+  };
+  from.forEach(deep_copy_xy);
```

Comment:
For cleanliness I don't think we need to extract the function out and we can inline it instead.  No big deal though